lib/screens/task_list_screen.dart

```diff
@@ -102,7 +106,7 @@ class _TaskListScreenState extends State<TaskListScreen> {
               ),
         title: Text(task.title),
         // ToDo Change subtitle to task description
-        subtitle: Text(task.title),
+        subtitle: Text(description),
```

Comment:
You could use the syntactic sugar of Dart like e.g.
```dart
subtitle: Text(task.description ?? ''),
```

* https://dart.dev/guides/language/language-tour#conditional-expressions